content/health.html

```diff
@@ -1,108 +1,148 @@
+  </span>
+  The cohort experienced a wide variety of mental health issues. Trying to
+  manage academics, co-op, and social lives can take a toll on us. It's
+  important to take some time to focus on ourselves and do the things that are
+  important and relaxing for each of us.
+</p>
+<div class="half second">
+  <span class="graph-title">What mental health issues have you faced?</span>
+  <div id="mental-health-issues"></div>
+</div>
 
 <p class="half">
   <span class="big">
-    Over half of respondents reported that mental health effected at least one school term.
+    Over half of respondents reported that mental health effected at least one
+    school term.
   </span>
-  Mental health can have large impacts on a student's life ranging from academic to social. 
-  Approximately, 10% of respondents reported mental health affecting half of their undergrad degree.
+  Mental health can have large impacts on a student's life ranging from academic
+  to social. Approximately, 10% of respondents reported mental health affecting
+  half of their undergrad degree.
 </p>
 <div class="half second">
-  <span class="graph-title">How many school terms were affected (academically, socially, or otherwise) by struggling with mental health?</span>
+  <span class="graph-title"
+    >How many school terms were affected (academically, socially, or otherwise)
+    by struggling with mental health?</span
+  >
   <div id="mental-health-terms"></div>
 </div>
 
 <p class="half">
   <span class="big">
-    University of Waterloo Counselling Services was the most popular mental health resource.
+    University of Waterloo Counselling Services was the most popular mental
+    health resource.
   </span>
-  While some respondents reported using some type of counselling service, many of those who struggled with mental health did not use any.
+  While some respondents reported using some type of counselling service, many
+  of those who struggled with mental health did not use any.
 </p>
 <div class="half second">
-  <span class="graph-title">Have you used any of the following counselling resources during your undergrad?</span>
+  <span class="graph-title"
+    >Have you used any of the following counselling resources during your
+    undergrad?</span
+  >
   <div id="councelling"></div>
 </div>
 
 <div class="vbx4"></div>
 
 <h3>Imposter Syndrome</h3>
 <p class="half">
-  <span class="big">Over 75% of respondents felt imposter syndrome over their university years.</span>
-  Impostor syndrome refers to an individual's internal experience of believing that they are not as competent as others perceive them to be. 
-  The Software Engineering program is very competitive and has one of the highest admission averages in Waterloo. This can place students in a 
-  pressure-to-achieve environment which might have led to students feeling like an imposter.
+  <span class="big"
+    >Over 75% of respondents felt imposter syndrome over their university
```

Comment:
Honestly I find this formatting a bit weird but it works? 🙃

Reply:
omg good catch - prettier just be wack